virtcontainers/pkg/cloud-hypervisor/client/go.mod

```diff
@@ -3,5 +3,7 @@ module github.com/GIT_USER_ID/GIT_REPO_ID
 require (
 	github.com/antihax/optional v1.0.0
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	
+
 )
+
```

Comment:
Why this change?

Reply:
Not sure about that one...I think it was an automatic editor change. I can revert the `go 1.13` line.